docs/source/model_doc/vit_mae.mdx

```diff
@@ -47,7 +47,8 @@ alt="drawing" width="600"/>
 
 <small> MAE architecture. Taken from the <a href="https://arxiv.org/abs/2111.06377">original paper.</a> </small>
 
-This model was contributed by [nielsr](https://huggingface.co/nielsr). The original code can be found [here](https://github.com/facebookresearch/mae).
+This model was contributed by [nielsr](https://huggingface.co/nielsr). TensorFlow version of the model was contributed by [sayakpaul](https://github.com/sayakpaul) and 
+[ariG23498](https://github.com/ariG23498) (equal contribution). The original code can be found [here](https://github.com/facebookresearch/mae). 
```

Comment:
We could also maybe link [our tutorial](https://keras.io/examples/vision/masked_image_modeling/) here and say something like:

> For a visual understanding of how MAEs work you can check out [this post](https://keras.io/examples/vision/masked_image_modeling/).

Reply:
Great idea! Will get to this.

Reply:
Cool. Including it under the **Tips** section would be the best IMO.